ratbagd/ratbagd-device.c

```diff
@@ -172,6 +172,8 @@ static void ratbagd_device_commit_pending(void *data)
 	int r;
 
 	r = ratbag_device_commit(device->lib_device);
+	if (r)
+		log_error("error committing device (%d)\n", r);
```

Comment:
Shouldn't it be `log_error(ctx,"...")`?

Reply:
Ah, yeah. Thanks for catching that.

Reply:
Actually no, not here. I just checked and we only use that for the drivers.

Reply:
Whoops yes, think I mixed something up.